Windows_IP: '192.168.1.23'
ServerPort: 8090
AuboIP: '192.168.1.115'
EE_DIS_TCP: 0.13
TBC: [-0.00780, -0.05902, 0.99823  ,0.33300401,
      -0.02279, 0.99801 , 0.05883  ,-0.45455141,
      -0.99971, -0.02229, -0.00913 ,0.68868421,
      0,0,0,1 ]
StartPoint: [-29.49778,27.381235,120.70059,87.6384,123.0182,-5.469]
StartPointTest: [6.914,57.896,147.5397,93.8056,87.2035,10.226]
StartPointXYZ_IN_TCP: [0.589118,-0.306314,0.815494]
Aubomaxacctuple: [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
Aubomaxvelctuple: [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
Aubominacctuple:  [0.1,0.1,0.1,0.1,0.1,0.1]
Aubominvelctuple: [0.1,0.1,0.1,0.1,0.1,0.1]